### PR TITLE
Remove unused method in DroidKaigi2023Day

### PR DESCRIPTION
**Pull request by fumiya-kume**
## Issue
- N/A

## Overview (Required)
- I guess it want to be `dynamic` ?

### CODE REVIEW

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt [hunk 1]
```diff
@@ -78,7 +78,7 @@ public enum class DroidKaigi2023Day(
             }
         }
 
-        public fun defaultDyamicThemeDate(): Boolean {
+        public fun defaultDynamicThemeDate(): Boolean {
             return Day1.start < Clock.System.now()
         }
```
Comment (takahirom) [inline core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt:83]: Oh we don't use this method for this year. Could you remove this?

Comment (fumiya-kume) [reply core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt:83]: Sure!

Comment (fumiya-kume) [reply core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt]: Changing in https://github.com/DroidKaigi/conference-app-2023/pull/1025/commits/ef613509ed234174f0c90695e05462ec457464ea

Also, Updated the PR title following the actual changes.

**Comment by takahirom** [comment]
Thank you for your cooperation!

**Review by takahirom — APPROVED** [review]

**Pull request merged**